lib/rexml/element.rb

```diff
@@ -1270,27 +1270,14 @@ def [](name_or_index)
     # With arguments +name+ and +namespace+ given,
     # returns the value of the named attribute if it exists, otherwise +nil+:
     #
-    #   xml_string = "<root xmlns:a='a' a:x='a:x' x='x'/>"
+    #   xml_string = "<root xmlns:a='http://example.com/a' a:x='a:x' x='x'/>"
     #   document = REXML::Document.new(xml_string)
     #   document.root.attribute("x")      # => x='x'
-    #   document.root.attribute("x", "a") # => a:x='a:x'
+    #   document.root.attribute("x", "http://example.com/a") # => a:x='a:x'
```

Comment:
I also updated here to make clear that the `namespace` argument takes a namespace URI.